cli/medperf/decorators.py

```diff
@@ -24,7 +24,6 @@ def wrapper(*args, **kwargs):
         except Exception as e:
             logging.error("An unexpected error occured. Terminating.")
             logging.error(e)
-            cleanup()
-            raise e
+            pretty_error("An unexpected error occured")
```

Comment:
missing `ui` :)

Reply:
I always forget that argument! Thanks for realizing!